integration-tests/minio/src/test/java/org/apache/camel/quarkus/component/minio/it/MinioTestResource.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.quarkus.component.minio.it;
+
+import java.time.Duration;
+import java.util.Map;
+
+import org.apache.camel.quarkus.testcontainers.ContainerResourceLifecycleManager;
+import org.apache.camel.util.CollectionHelper;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.wait.strategy.HttpWaitStrategy;
+
+public class MinioTestResource implements ContainerResourceLifecycleManager {
+
+    public static final String CONTAINER_ACCESS_KEY = "MINIO_ACCESS_KEY";
+    public static final String CONTAINER_SECRET_KEY = "MINIO_SECRET_KEY";
+    private final String CONTAINER_IMAGE = "minio/minio:latest";
```

Comment:
Can we avoid the `latest` tag and pin to something else?

Reply:
I've replaced latest with the current tag.